core/spec/lib/spree/core/testing_support/preferences_spec.rb

```diff
@@ -4,7 +4,23 @@
 
 RSpec.describe Spree::TestingSupport::Preferences do
   describe 'resetting the app configuration' do
+    around do |example|
+      with_unfrozen_spree_preference_store do
+        behavior = Spree::Deprecation.behavior
```

Comment:
I think you should use  `Spree::Deprecation.silence {}` instead here, right?

Reply:
@kennyadsl 👍 that's much better, thank you!